main.go

```diff
@@ -120,12 +123,38 @@ func sendEmail(event *types.Event) error {
 		return bodyErr
 	}
 
-	msg := []byte("To: " + toEmail + "\r\n" +
-		"Subject: " + subject + "\r\n" +
-		"\r\n" +
-		body + "\r\n")
+	if insecure {
+		msg := "To: " + toEmail + "\r\n" +
+			"Subject: " + subject + "\r\n" +
+			"\r\n" +
+			body + "\r\n"
+		smtpconn, connErr := smtp.Dial(smtpAddress)
+		if connErr != nil {
+			return connErr
+		}
+		defer smtpconn.Close()
+		smtpconn.Mail(fromEmail)
+		smtpconn.Rcpt(toEmail)
+		smtpdata, dataErr := smtpconn.Data()
+		if dataErr != nil {
+			return dataErr
+		}
+		defer smtpdata.Close()
+		buf := bytes.NewBufferString(msg)
+		if _, dataErr = buf.WriteTo(smtpdata); dataErr != nil {
+			return dataErr
+		}
+
+		return nil
+	} else {
```

Comment:
Since the if block returns before the else block is reached, surrounding this code with an else statement is unnecessary: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
This is the if block for testing the boolean insecure which if false falls into the else clause of using the secure connection.  Or am I missing something else here?

Reply:
Updated to remove the unnecessary else.